addScript.js

```diff
@@ -3,8 +3,15 @@
 	Author Tobias Koppers @sokra
 */
 module.exports = function(src) {
-	if (typeof execScript !== "undefined")
-		execScript(src);
-	else
-		eval.call(null, src);
+	try {
+		if (typeof eval !== "undefined") {
+			eval.call(null, src);
+		} else if (typeof execScript !== "undefined") {
+			execScript(src);
+		} else {
+			// TODO error: no eval function available
```

Comment:
Using `Todo` in code is bad practice. How we can solve this problem, what you think?